Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM openjdk:8-jre-alpine
+COPY /target/loofah-graph-api.jar .
+CMD /usr/bin/java -Dspring.data.mongodb.uri=mongodb://loofah-database:27017/test -jar ./loofah-graph-api.jar
```

Comment:
Feels like this dockerfile should have a mvn package or install command before copying the jar file? since the `/target` folder is only created after doing a `mvn clean install`

Reply:
hmm this made me ponder. I think I'd have to copy the entire project onto the container to run `mvn clean install` from the dockerfile. On all the dockerised apps I've worked on, we usually run `mvn clean install` to create the jar file locally, copy that onto the container, then run (pseudo-command) `docker run --build`. The `runLoofah.sh` script includes the `mvn clean install` command if that helps?

Reply:
ah yeah you are right, looking at a Dockerfile I've used on a previous project with a java component, we have had to copy the whole project first before running the `mvn clean package` command: 
```
COPY pom.xml .
RUN mvn -B dependency:go-offline
COPY . .
RUN mvn -B clean package -Dmaven.test.skip=true
```

What you've described makes sense though so no need to change 👍 Thanksss